### PR TITLE
Pyronear fork

### PR DESCRIPTION
**Pull request by maevaportfolio**

### CODE REVIEW

**Comment by maevaportfolio** [comment]
Hello Gaetan,
I hope you're doing well :)

You'll find two major modifications:

- The first modification is in the file processing.py: I removed redundant imports and unused variables and added comments and docstrings for better readability.
- The second modification is the addition of the file test_process_function that allows verification of the correctness of each function.

Can you please review my code and tell me if I corrected it well?
Thank you.

Have a great day!

Best regards,
Maeva N'GUESSAN

File: super_resolution/ESRGAN_pretrained/images/output.png
Comment (gaetanbrison) [inline super_resolution/ESRGAN_pretrained/images/output.png:1]: really like the approach where you showcase the 3 results

File: image_preprocessing/test_process_functions.py
Comment (gaetanbrison) [inline image_preprocessing/test_process_functions.py:1]: Don't hesitate to add some documentation on methods in numpy documentation style :)

File: image_preprocessing/processing2.py
Comment (gaetanbrison) [inline image_preprocessing/processing2.py:1]: Great! way to go 🚀

**Review by gaetanbrison — APPROVED** [review]
Great pull request!
I really like the image with the 3 results aligned.
Don't hesitate to add some numpy documentation style  on the script "test_process_functions.py"

🚀

**Pull request merged**